aether-odk-module/aether/odk/api/xform_utils.py

```diff
@@ -636,6 +646,20 @@ def __get_xform_instance_skeleton(xml_definition):
                 xpath = bind_entry.get('@nodeset')
                 schema[xpath]['type'] = bind_entry.get('@type')
                 schema[xpath]['required'] = bind_entry.get('@required') == 'true()'
+                select_options = None
+                if schema[xpath]['type'] in SELECT_TAGS:
+                    select_node = list(__find_by_key_value(xform_dict, '@ref', xpath))[0]
+                    select_options = list(select_node.get('item', []))
+                    # limitation: skips selects linked to a datasource with 'itemset'
+                    # todo: extend visualization decoration to itemsets
+                    for option in select_options:
+                        if 'label' in option and isinstance(option['label'], dict) \
+                                and '@ref' in option['label'] and option['label']['@ref'].startswith('jr:itext'):
+                            ref_id = option['label']['@ref'].split("'")[1]
+                            translated_label = itexts[ref_id]
+                            option['label'] = translated_label
+                    if select_options and len(select_options) <= 20:
```

Comment:
Could you change this magic number `20` with a constant like `SELECT_CHOICES_CUTOFF` of whatever you like most?